### ISSUE TITLE
RFE: please ensure dmidecode -t system shows proper values

### ISSUE THREAD

**Issue by pcfe** [IceWhaleTech/ZimaBoard-BIOS#6]
# request for enhancement

As a user of Red-Hat-flavoured distros (Fedora, CentOS Stream, RHEL) it is a quality of life feature if a machine's DMI is filled correctly. Meaning that while nothing stops working if it has for example `Manufacturer: Default string` under type System and it is only a minor annoyance to work around a UUID that is identical in multiple boards, but this is not a nice user experience. Hence this RFE.

## Steps to reproduce

1. install an operating system of the RedHat family, e.g. CentOS Stream 8 (tested with all updates that are available today applied) or Fedora 38 (tested with Fedora-KDE-Live-x86_64-38-1.6.iso)
2. run `dmidecode`

## Actual result and requested enhancements

tl;dr: most tools I use display info related to _type system_, not _baseboard_. Please fill type System too.

Multiple fields have generic values or values that do not match what is shown in Setup, it would be a quality of life increase if as many as possible of these could be filled, by order of preference:

1. comparable BIOS version strings of the version when checking your download site versus `dmidecode -t bios`
2. a UUID that is unique for each purchased board. 
3. Finding the string _ZimaBoard 832_ when looking at `dmidecode -t system`
4. a Serial number that is unique for each purchased board. 
5. all the other generic strings "to be filled by OEM", "default", "other", etc fixed

#### from `dmidecode -t bios`, I get

```text
BIOS Information
        Vendor: American Megatrends Inc.
        Version: 5.12
        Release Date: 07/26/2022
[…]
        BIOS Revision: 5.12
```

#### from firmware Setup screens, I get

```text
Project Version: ZMB V1.0
Build Date and Time: 07/26/2022 12:09:34
```

#### Comparing to what is offered as BIOS download

On your BIOS download site at <https://github.com/IceWhaleTech/ZimaBoard-BIOS/releases> I see

N06
Mar 4, 2022

which per date seems older than what was shipped on my ZimaBoard 832. (so I did not apply N06 to my board)

#### Wished state

Consistency across all 3 (BIOS release downloads, BIOS Setup screen, dmidecode -t bios).

Ideally, if an unreleased N09 from May 07th 2023 were flashed but N09 was still using AMI's 5.12 code, then such an example would show the following in `dmidecode -t bios`:
- Version: N09
- Release Date: 05/07/2023
- BIOS Revision: 5.12

### System Information

I am aware that `dmidecode -t baseboard` has some data filled (see below), but most of the tooling I use (for the machines which I have installed with CentOS Stream, RHEL or Fedora) queries type system from DMI.

(as an aside, when physically looking for e.g. a failed node in a mixed cluster, then I think of the whole system too, baseplate and radiator attached, not just the naked motherboard. If the whole unit had a sticker with a unique serial that would make identifying one of many ZimaBorads, provided DMI has the same value for serial, even easier)

```bash
[root@zimaboard-01 ~]# dmidecode -t system
# dmidecode 3.3
Getting SMBIOS data from sysfs.
SMBIOS 3.0.0 present.

Handle 0x0001, DMI type 1, 27 bytes
System Information
	Manufacturer: Default string
	Product Name: Default string
	Version: Default string
	Serial Number: Default string
	UUID: 03000200-0400-0500-0006-000700080009
	Wake-up Type: Power Switch
	SKU Number: Default string
	Family: Default string

Handle 0x0022, DMI type 12, 5 bytes
System Configuration Options
	Option 1: Default string

Handle 0x0027, DMI type 32, 20 bytes
System Boot Information
	Status: No errors detected

```

#### Wished state

- Manufacturer, Product Name, Version, SKU Number: correctly filled and identical across all shipped SimaBoard of a model (832, 632, etc)
- UUID and Serial Number: different for each physical board

### Chassis Information

this one is minor, but being able to see the SKU would be helpful the day I want to order more identical ones.

### Baseboard Information

A (unique per physical ZimaBoard 832) serial number would be nice.
and properly naming the entries that now read 'to be filled by OEM' or 'other' would be appreciated

```bash
[root@zimaboard-01 ~]# dmidecode -t baseboard
# dmidecode 3.3
Getting SMBIOS data from sysfs.
SMBIOS 3.0.0 present.

Handle 0x0002, DMI type 2, 15 bytes
Base Board Information
	Manufacturer: IceWhale Technology Co.,Ltd.
	Product Name: ZimaBoard 832
	Version: ZMB 1.0
	Serial Number: Default string
	Asset Tag: Default string
	Features:
		Board is a hosting board
		Board is replaceable
	Location In Chassis: Default string
	Chassis Handle: 0x0003
	Type: Motherboard
	Contained Object Handles: 0

Handle 0x0020, DMI type 10, 6 bytes
On Board Device Information
	Type: Video
	Status: Enabled
	Description:    To Be Filled By O.E.M.

Handle 0x0031, DMI type 41, 11 bytes
Onboard Device
	Reference Designation: Onboard - Ethernet
	Type: Ethernet
	Status: Disabled
	Type Instance: 1
	Bus Address: 0000:ff:00.0

Handle 0x0032, DMI type 41, 11 bytes
Onboard Device
	Reference Designation: Onboard - Other
	Type: Other
	Status: Enabled
	Type Instance: 1
	Bus Address: 0000:00:00.0

Handle 0x0033, DMI type 41, 11 bytes
Onboard Device
	Reference Designation: Onboard - Video
	Type: Video
	Status: Enabled
	Type Instance: 1
	Bus Address: 0000:00:02.0

Handle 0x0034, DMI type 41, 11 bytes
Onboard Device
	Reference Designation: Onboard - Other
	Type: Other
	Status: Enabled
	Type Instance: 2
	Bus Address: 0000:00:0d.0

Handle 0x0035, DMI type 41, 11 bytes
Onboard Device
	Reference Designation: Onboard - Other
	Type: Other
	Status: Enabled
	Type Instance: 3
	Bus Address: 0000:00:0d.2

Handle 0x0036, DMI type 41, 11 bytes
Onboard Device
	Reference Designation: Onboard - Other
	Type: Other
	Status: Enabled
	Type Instance: 4
	Bus Address: 0000:00:0d.3

Handle 0x0037, DMI type 41, 11 bytes
Onboard Device
	Reference Designation: Onboard - Sound
	Type: Sound
	Status: Enabled
	Type Instance: 1
	Bus Address: 0000:00:0e.0

Handle 0x0038, DMI type 41, 11 bytes
Onboard Device
	Reference Designation: Onboard - Other
	Type: Other
	Status: Enabled
	Type Instance: 5
	Bus Address: 0000:00:0f.0

Handle 0x0039, DMI type 41, 11 bytes
Onboard Device
	Reference Designation: Onboard - SATA
	Type: SATA Controller
	Status: Enabled
	Type Instance: 1
	Bus Address: 0000:00:12.0

Handle 0x003A, DMI type 41, 11 bytes
Onboard Device
	Reference Designation: Onboard - Other
	Type: Other
	Status: Enabled
	Type Instance: 6
	Bus Address: 0000:00:15.0

Handle 0x003B, DMI type 41, 11 bytes
Onboard Device
	Reference Designation: Onboard - Other
	Type: Other
	Status: Enabled
	Type Instance: 7
	Bus Address: 0000:00:1c.0

Handle 0x003C, DMI type 41, 11 bytes
Onboard Device
	Reference Designation: Onboard - Other
	Type: Other
	Status: Enabled
	Type Instance: 8
	Bus Address: 0000:00:1f.0

Handle 0x003D, DMI type 41, 11 bytes
Onboard Device
	Reference Designation: Onboard - Other
	Type: Other
	Status: Enabled
	Type Instance: 9
	Bus Address: 0000:00:1f.1

```

## Examples of things that default to type system for el8 hosts

here are a few examples where users will encounter _default string_ or _to be filled mby OEM_ or a UUID that does not really seem to be unique for each physical board.

### Foreman (aka Red Hat Satellite)

keeps 2 hosts apart by their UUID. Sure, I can override the `03000200-0400-0500-0006-000700080009`, it's [well documented ](https://theforeman.org/plugins/katello/nightly/troubleshooting/content_hosts.html), but it is less hassle if out of the box each physical system has a unique UUID.

### Ceph (aka Red hat Ceph Storage)

Modern versions of Ceph (I currently run Pacific, not sure how many versions back the info appeared in the Dashboard)  will show in the dashboard under Hosts, in the column 'Model', what is available under
System Information. It would be nice to read there something along the lines of _ZimaBoard 832_ rather then Default string

###  lshw in GUI mode

![Screenshot from 2023-05-07 12-38-42](https://user-images.githubusercontent.com/3717066/236674405-ab639f79-bde9-44e5-bdf5-a65c381fe5ea.png)

### lshw in CLI mode

Here it would be nice if the first block had correct values instead of Default String

```bash
[root@zimaboard-01 ~]# lshw
zimaboard-01.internal.pcfe.net
    description: Desktop Computer
    product: Default string (Default string)
    vendor: Default string
    version: Default string
    serial: Default string
    width: 64 bits
    capabilities: smbios-3.0.0 dmi-3.0.0 smp vsyscall32
    configuration: boot=normal chassis=desktop family=Default string sku=Default string uuid=03000200-0400-0500-0006-000700080009
  *-core
       description: Motherboard
       product: ZimaBoard 832
       vendor: IceWhale Technology Co.,Ltd.
       physical id: 0
       version: ZMB 1.0
       serial: Default string
       slot: Default string
     *-firmware
          description: BIOS
          vendor: American Megatrends Inc.
          physical id: 0
          version: 5.12
          date: 07/26/2022
          size: 64KiB
          capacity: 5MiB
          capabilities: pci upgrade shadowing cdboot bootselect socketedrom edd int13floppy1200 int13floppy720 int13floppy2880 int5printscreen int9keyboard int14serial int17printer acpi usb biosbootspecification uefi
     *-memory
          description: System Memory
          physical id: 28
          slot: System board or motherboard
          size: 8GiB
        *-bank:0
             description: DIMM LPDDR4 Synchronous 1600 MHz (0.6 ns)
             product: 123456789012345678
             vendor: ABCD
             physical id: 0
             serial: 1234
             slot: ChannelA-DIMM0
             size: 2GiB
             width: 16 bits
             clock: 1600MHz (0.6ns)
        *-bank:1
             description: DIMM LPDDR4 Synchronous 1600 MHz (0.6 ns)
             product: 123456789012345678
             vendor: ABCD
             physical id: 1
             serial: 1234
             slot: ChannelB-DIMM0
             size: 2GiB
             width: 16 bits
             clock: 1600MHz (0.6ns)
        *-bank:2
             description: DIMM LPDDR4 Synchronous 1600 MHz (0.6 ns)
             product: 123456789012345678
             vendor: ABCD
             physical id: 2
             serial: 1234
             slot: ChannelC-DIMM0
             size: 2GiB
             width: 16 bits
             clock: 1600MHz (0.6ns)
        *-bank:3
             description: DIMM LPDDR4 Synchronous 1600 MHz (0.6 ns)
             product: 123456789012345678
             vendor: ABCD
             physical id: 3
             serial: 1234
             slot: ChannelD-DIMM0
             size: 2GiB
             width: 16 bits
             clock: 1600MHz (0.6ns)
     *-cache:0
          description: L1 cache
          physical id: 2e
          slot: CPU Internal L1
          size: 224KiB
          capacity: 224KiB
          capabilities: synchronous internal write-back
          configuration: level=1
     *-cache:1
          description: L2 cache
          physical id: 2f
          slot: CPU Internal L2
          size: 2MiB
          capacity: 2MiB
          capabilities: synchronous internal write-back unified
          configuration: level=2
     *-cpu
          description: CPU
          product: Intel(R) Celeron(R) CPU N3450 @ 1.10GHz
          vendor: Intel Corp.
          physical id: 30
          bus info: cpu@0
          version: 6.92.9
          slot: SOCKET 0
          size: 699MHz
          capacity: 2400MHz
          width: 64 bits
          clock: 100MHz
          capabilities: lm fpu fpu_exception wp vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp x86-64 constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology tsc_reliable nonstop_tsc cpuid aperfmperf tsc_known_freq pni pclmulqdq dtes64 monitor ds_cpl vmx est tm2 ssse3 sdbg cx16 xtpr pdcm sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave rdrand lahf_lm 3dnowprefetch cpuid_fault cat_l2 ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust smep erms mpx rdt_a rdseed smap clflushopt intel_pt sha_ni xsaveopt xsavec xgetbv1 xsaves dtherm ida arat pln pts md_clear arch_capabilities cpufreq
          configuration: cores=4 enabledcores=4 microcode=72 threads=4
     *-pci
          description: Host bridge
          product: Celeron N3350/Pentium N4200/Atom E3900 Series Host Bridge
          vendor: Intel Corporation
          physical id: 100
          bus info: pci@0000:00:00.0
          version: 0b
          width: 32 bits
          clock: 33MHz
        *-display
             description: VGA compatible controller
             product: HD Graphics 500
             vendor: Intel Corporation
             physical id: 2
             bus info: pci@0000:00:02.0
             logical name: /dev/fb0
             version: 0b
             width: 64 bits
             clock: 33MHz
             capabilities: pciexpress msi pm vga_controller bus_master cap_list fb
             configuration: depth=32 driver=i915 latency=0 resolution=1920,1080
             resources: irq:127 memory:90000000-90ffffff memory:80000000-8fffffff ioport:f000(size=64) memory:c0000-dffff
        *-multimedia
             description: Audio device
             product: Celeron N3350/Pentium N4200/Atom E3900 Series Audio Cluster
             vendor: Intel Corporation
             physical id: e
             bus info: pci@0000:00:0e.0
             logical name: card0
             logical name: /dev/snd/controlC0
             logical name: /dev/snd/hwC0D2
             logical name: /dev/snd/pcmC0D3p
             logical name: /dev/snd/pcmC0D7p
             logical name: /dev/snd/pcmC0D8p
             version: 0b
             width: 64 bits
             clock: 33MHz
             capabilities: pm msi pciexpress bus_master cap_list
             configuration: driver=snd_hda_intel latency=0
             resources: irq:129 memory:91310000-91313fff memory:91000000-910fffff
           *-input:0
                product: HDA Intel PCH HDMI/DP,pcm=7
                physical id: 0
                logical name: input10
                logical name: /dev/input/event7
           *-input:1
                product: HDA Intel PCH HDMI/DP,pcm=8
                physical id: 1
                logical name: input11
                logical name: /dev/input/event8
           *-input:2
                product: HDA Intel PCH HDMI/DP,pcm=3
                physical id: 2
                logical name: input9
                logical name: /dev/input/event6
        *-communication
             description: Communication controller
             product: Celeron N3350/Pentium N4200/Atom E3900 Series Trusted Execution Engine
             vendor: Intel Corporation
             physical id: f
             bus info: pci@0000:00:0f.0
             version: 0b
             width: 64 bits
             clock: 33MHz
             capabilities: pm msi bus_master cap_list
             configuration: driver=mei_me latency=0
             resources: irq:128 memory:9131d000-9131dfff
        *-sata
             description: SATA controller
             product: Celeron N3350/Pentium N4200/Atom E3900 Series SATA AHCI Controller
             vendor: Intel Corporation
             physical id: 12
             bus info: pci@0000:00:12.0
             version: 0b
             width: 32 bits
             clock: 66MHz
             capabilities: sata msi pm ahci_1.0 bus_master cap_list
             configuration: driver=ahci latency=0
             resources: irq:126 memory:91314000-91315fff memory:9131a000-9131a0ff ioport:f090(size=8) ioport:f080(size=4) ioport:f060(size=32) memory:91319000-913197ff
        *-pci:0
             description: PCI bridge
             product: Celeron N3350/Pentium N4200/Atom E3900 Series PCI Express Port A #1
             vendor: Intel Corporation
             physical id: 13
             bus info: pci@0000:00:13.0
             version: fb
             width: 32 bits
             clock: 33MHz
             capabilities: pci pciexpress msi pm normal_decode bus_master cap_list
             configuration: driver=pcieport
             resources: irq:120
        *-pci:1
             description: PCI bridge
             product: Celeron N3350/Pentium N4200/Atom E3900 Series PCI Express Port B #1
             vendor: Intel Corporation
             physical id: 14
             bus info: pci@0000:00:14.0
             version: fb
             width: 32 bits
             clock: 33MHz
             capabilities: pci pciexpress msi pm normal_decode bus_master cap_list
             configuration: driver=pcieport
             resources: irq:121 ioport:e000(size=4096) memory:91200000-912fffff
           *-network
                description: Ethernet interface
                product: RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller
                vendor: Realtek Semiconductor Co., Ltd.
                physical id: 0
                bus info: pci@0000:02:00.0
                logical name: enp2s0
                version: 15
                serial: 00:e0:4c:19:77:49
                size: 1Gbit/s
                capacity: 1Gbit/s
                width: 64 bits
                clock: 33MHz
                capabilities: pm msi pciexpress msix bus_master cap_list ethernet physical tp mii 10bt 10bt-fd 100bt 100bt-fd 1000bt-fd autonegotiation
                configuration: autonegotiation=on broadcast=yes driver=r8169 driverversion=4.18.0-489.el8.x86_64 duplex=full firmware=rtl8168h-2_0.0.2 02/26/15 ip=192.168.50.115 latency=0 link=yes multicast=yes port=MII speed=1Gbit/s
                resources: irq:22 ioport:e000(size=256) memory:91204000-91204fff memory:91200000-91203fff
        *-pci:2
             description: PCI bridge
             product: Celeron N3350/Pentium N4200/Atom E3900 Series PCI Express Port B #2
             vendor: Intel Corporation
             physical id: 14.1
             bus info: pci@0000:00:14.1
             version: fb
             width: 32 bits
             clock: 33MHz
             capabilities: pci pciexpress msi pm normal_decode bus_master cap_list
             configuration: driver=pcieport
             resources: irq:122 ioport:d000(size=4096) memory:91100000-911fffff
           *-network
                description: Ethernet interface
                product: RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller
                vendor: Realtek Semiconductor Co., Ltd.
                physical id: 0
                bus info: pci@0000:03:00.0
                logical name: enp3s0
                version: 15
                serial: 00:e0:4c:19:77:4b
                capacity: 1Gbit/s
                width: 64 bits
                clock: 33MHz
                capabilities: pm msi pciexpress msix bus_master cap_list ethernet physical tp mii 10bt 10bt-fd 100bt 100bt-fd 1000bt-fd autonegotiation
                configuration: autonegotiation=on broadcast=yes driver=r8169 driverversion=4.18.0-489.el8.x86_64 firmware=rtl8168h-2_0.0.2 02/26/15 latency=0 link=no multicast=yes port=MII
                resources: irq:23 ioport:d000(size=256) memory:91104000-91104fff memory:91100000-91103fff
        *-usb
             description: USB controller
             product: Celeron N3350/Pentium N4200/Atom E3900 Series USB xHCI
             vendor: Intel Corporation
             physical id: 15
             bus info: pci@0000:00:15.0
             version: 0b
             width: 64 bits
             clock: 33MHz
             capabilities: pm msi xhci bus_master cap_list
             configuration: driver=xhci_hcd latency=0
             resources: irq:123 memory:91300000-9130ffff
           *-usbhost:0
                product: xHCI Host Controller
                vendor: Linux 4.18.0-489.el8.x86_64 xhci-hcd
                physical id: 0
                bus info: usb@1
                logical name: usb1
                version: 4.18
                capabilities: usb-2.00
                configuration: driver=hub slots=8 speed=480Mbit/s
              *-usb
                   description: USB hub
                   product: USB2.0 Hub
                   vendor: VIA Labs, Inc.
                   physical id: 2
                   bus info: usb@1:2
                   version: 90.11
                   capabilities: usb-2.10
                   configuration: driver=hub slots=4 speed=480Mbit/s
                 *-usb:0
                      description: Keyboard
                      product: HID 046a:0023
                      vendor: Cherry GmbH
                      physical id: 3
                      bus info: usb@1:2.3
                      logical name: input5
                      logical name: /dev/input/event2
                      logical name: input5::capslock
                      logical name: input5::numlock
                      logical name: input5::scrolllock
                      logical name: input6
                      logical name: /dev/input/event3
                      version: 2.20
                      capabilities: usb-2.00 usb
                      configuration: driver=usbhid maxpower=100mA speed=2Mbit/s
                 *-usb:1
                      description: Mouse
                      product: Logitech USB Laser Mouse
                      vendor: Logitech
                      physical id: 4
                      bus info: usb@1:2.4
                      logical name: input4
                      logical name: /dev/input/event1
                      logical name: /dev/input/mouse0
                      version: 56.01
                      capabilities: usb-2.00 usb
                      configuration: driver=usbhid maxpower=98mA speed=2Mbit/s
           *-usbhost:1
                product: xHCI Host Controller
                vendor: Linux 4.18.0-489.el8.x86_64 xhci-hcd
                physical id: 1
                bus info: usb@2
                logical name: usb2
                version: 4.18
                capabilities: usb-3.00
                configuration: driver=hub slots=7 speed=5000Mbit/s
              *-usb:0
                   description: Mass storage device
                   product: NVME USB3.2
                   vendor: JMicron
                   physical id: 1
                   bus info: usb@2:1
                   logical name: scsi0
                   version: 2.12
                   serial: 0123456789ABC
                   capabilities: usb-3.20 scsi
                   configuration: driver=uas maxpower=896mA speed=5000Mbit/s
                 *-disk
                      description: SCSI Disk
                      product: 3.2
                      vendor: NVME USB
                      physical id: 0.0.0
                      bus info: scsi@0:0.0.0
                      logical name: /dev/sda
                      version: 0212
                      serial: 0123456789ABC
                      size: 238GiB (256GB)
                      capabilities: gpt-1.00 partitioned partitioned:gpt
                      configuration: ansiversion=6 guid=62994bfd-50f5-479f-b40c-53c12150d533 logicalsectorsize=512 sectorsize=4096
                    *-volume:0 UNCLAIMED
                         description: Windows FAT volume
                         vendor: mkfs.fat
                         physical id: 1
                         bus info: scsi@0:0.0.0,1
                         version: FAT32
                         serial: 9d5b-b2c8
                         size: 598MiB
                         capacity: 599MiB
                         capabilities: boot fat initialized
                         configuration: FATs=2 filesystem=fat name=EFI System Partition
                    *-volume:1
                         description: EFI partition
                         physical id: 2
                         bus info: scsi@0:0.0.0,2
                         logical name: /dev/sda2
                         logical name: /boot
                         serial: 88a716ea-bae7-4514-a08b-0c3cc6d99979
                         capacity: 1023MiB
                         configuration: mount.fstype=xfs mount.options=rw,seclabel,relatime,attr2,inode64,logbufs=8,logbsize=32k,noquota state=mounted
                    *-volume:2
                         description: LVM Physical Volume
                         vendor: Linux
                         physical id: 3
                         bus info: scsi@0:0.0.0,3
                         logical name: /dev/sda3
                         serial: sU7tRp-RtZi-Pf1L-yUDs-C3Py-7a6x-JT7EOi
                         size: 236GiB
                         capabilities: multi lvm2
              *-usb:1
                   description: USB hub
                   product: USB3.0 Hub
                   vendor: VIA Labs, Inc.
                   physical id: 2
                   bus info: usb@2:2
                   version: 90.11
                   capabilities: usb-3.00
                   configuration: driver=hub slots=4 speed=5000Mbit/s
        *-generic
             description: MMC Host
             product: Celeron N3350/Pentium N4200/Atom E3900 Series eMMC Controller
             vendor: Intel Corporation
             physical id: 1c
             bus info: pci@0000:00:1c.0
             logical name: mmc0
             version: 0b
             width: 64 bits
             clock: 33MHz
             capabilities: pm bus_master cap_list
             configuration: driver=sdhci-pci latency=0
             resources: irq:39 memory:91318000-91318fff memory:91317000-91317fff
           *-device
                description: SD/MMC Device
                product: DV4032
                vendor: Unknown (69)
                physical id: 1
                bus info: mmc@0:0001
                date: 09/2022
                serial: 1474301511
                capabilities: mmc
              *-interface:0
                   physical id: 1
                   logical name: /dev/mmcblk0rpmb
              *-interface:1
                   physical id: 2
                   logical name: /dev/mmcblk0
                   size: 31268536320
                   capabilities: gpt-1.00 partitioned partitioned:gpt
                   configuration: guid=002eb32e-ea04-4a34-8b17-240303106a2e logicalsectorsize=512 sectorsize=512
                 *-volume:0
                      description: Windows FAT volume
                      vendor: mkfs.fat
                      physical id: 1
                      logical name: /dev/mmcblk0p1
                      version: FAT32
                      serial: f087-0e41
                      size: 510MiB
                      capacity: 511MiB
                      capabilities: boot fat initialized
                      configuration: FATs=2 filesystem=fat
                 *-volume:1
                      description: EXT4 volume
                      vendor: Linux
                      physical id: 2
                      logical name: /dev/mmcblk0p2
                      version: 1.0
                      serial: 02a607c6-b697-4f13-a5a1-b779f868a988
                      size: 27GiB
                      capabilities: journaled extended_attributes large_files huge_files dir_nlink 64bit extents ext4 ext2 initialized
                      configuration: created=2021-12-20 07:51:13 filesystem=ext4 lastmountpoint=/run/media/testuser/02a607c6-b697-4f13-a5a1-b779f868a988 modified=2023-05-07 12:38:52 mounted=2023-05-07 11:17:09 state=clean
                 *-volume:2
                      description: Linux swap volume
                      vendor: Linux
                      physical id: 3
                      logical name: /dev/mmcblk0p3
                      version: 1
                      serial: 65ef69be-05ac-469a-929f-1e01e69c278b
                      size: 976MiB
                      capacity: 976MiB
                      capabilities: nofs swap initialized
                      configuration: filesystem=swap pagesize=4096
        *-isa
             description: ISA bridge
             product: Celeron N3350/Pentium N4200/Atom E3900 Series Low Pin Count Interface
             vendor: Intel Corporation
             physical id: 1f
             bus info: pci@0000:00:1f.0
             version: 0b
             width: 32 bits
             clock: 33MHz
             capabilities: isa bus_master
             configuration: driver=lpc_ich latency=0
             resources: irq:0
           *-pnp00:00
                product: PnP device PNP0c02
                physical id: 0
                capabilities: pnp
                configuration: driver=system
           *-pnp00:01
                product: PnP device PNP0303
                physical id: 1
                capabilities: pnp
                configuration: driver=i8042 kbd
           *-pnp00:02
                product: PnP device PNP0f13
                physical id: 2
                capabilities: pnp
                configuration: driver=i8042 aux
           *-pnp00:03
                product: PnP device PNP0c02
                physical id: 3
                capabilities: pnp
                configuration: driver=system
           *-pnp00:04
                product: PnP device PNP0c02
                physical id: 4
                capabilities: pnp
                configuration: driver=system
           *-pnp00:05
                product: PnP device PNP0b00
                physical id: 5
                capabilities: pnp
                configuration: driver=rtc_cmos
        *-serial
             description: SMBus
             product: Celeron N3350/Pentium N4200/Atom E3900 Series SMBus Controller
             vendor: Intel Corporation
             physical id: 1f.1
             bus info: pci@0000:00:1f.1
             version: 0b
             width: 64 bits
             clock: 33MHz
             configuration: driver=i801_smbus latency=0
             resources: irq:20 memory:91316000-913160ff ioport:f040(size=32)
  *-input:0
       product: Power Button
       physical id: 1
       logical name: input0
       logical name: /dev/input/event0
       capabilities: platform
  *-input:1
       product: Video Bus
       physical id: 2
       logical name: input7
       logical name: /dev/input/event4
       capabilities: platform
  *-input:2
       product: PC Speaker
       physical id: 3
       logical name: input8
       logical name: /dev/input/event5
       capabilities: isa
[root@zimaboard-01 ~]# 
```
